widget/src/svelte.d.ts

```diff
@@ -0,0 +1,25 @@
+declare module '*.svelte' {
```

Comment:
Is any of this file necessary? Did you follow a tutorial that suggested creating this, or was it necessary to create by hand?